src/code_gen.rs

```diff
@@ -505,14 +505,15 @@ std::string {cpp_var_id} = node_ptr->properties.at({parts});",
         let key_value_block = match func.udf_type {
             UdfType::Scalar => {
                 format!(
-                "                    let mut file = OpenOptions::new().append(true).open(\"result.txt\").unwrap();
+                "                    let mut file = OpenOptions::new().create(true).write(true).open(\"result.txt\").unwrap();
```

Comment:
This seems like a strange way to open a file for writing. What is the reason?

Reply:
I want to append to the file, not just open it and write over what's already there.  It seemed like this was the canonical way of doing that in rust.

Reply:
Rust is really strange sometimes.